### PR TITLE
Add default country to channel

### PR DESCRIPTION
**Pull request by maarcingebala**
Add default country to channel. Use the channel's default country for stock and tax calculations when country code is yet unknown in the checkout or product listings.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4190 Add to cart issues with warehouse outside of shipping zone](https://app.clickup.com/t/2549495/SALEOR-4190)

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
@krzysztofwolski Please review the descriptions, I'll be happy to improve them if they're not clear enough.

File: saleor/graphql/channel/mutations.py [hunk 1]
```diff
@@ -28,6 +29,14 @@ class ChannelCreateInput(ChannelInput):
     currency_code = graphene.String(
         description="Currency of the channel.", required=True
     )
+    default_country = CountryCodeEnum(
+        description=(
+            "Default country for the channel. Default country can be used in checkout "
```
Comment (krzysztofwolski) [inline saleor/graphql/channel/mutations.py:34]: 👌

**Review by krzysztofwolski — APPROVED** [review]

**Review by korycins — APPROVED** [review]
What about changelog? Would be good to have a note, that channel will receive a country field and all existing channels will receive automatic assigment

**Comment by maarcingebala** [comment]
@korycins I'll mention that in the release notes, but not sure if it should go to 3.0 changelog - it's part of the multichannel feature that this version introduces.

**Review by tomaszszymanski129 — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Pull request merged**